docs/sdk/script/fn/GetPlrViewMode.xml

```diff
@@ -21,17 +21,18 @@
     <desc>
       Determines the current view mode of a player. The function returns the following values: 
       <ul>
-        <li>0: Cursor - The player view follows the currently selected clonk (see <funclink>GetCursor</funclink>).</li>
-        <li>1: Target - The player view follows the object set by <funclink>SetPlrView</funclink>. Also see <funclink>GetPlrView</funclink>.</li>
-        <li>2: Scroll - The player is scrolling his view using the mouse.</li>
+        <li>C4PVM_Cursor = 0: Cursor - The player view follows the currently selected clonk (see <funclink>GetCursor</funclink>, <funclink>SetViewCursor</funclink>).</li>
+        <li>C4PVM_Target = 1: Target - The player view follows the object set by <funclink>SetPlrView</funclink>. Also see <funclink>GetPlrView</funclink>.</li>
+        <li>C4PVM_Scrolling = 2: Scroll - The player is scrolling his view using the mouse.</li>
```

Comment:
If this line is touched anyway, I'd suggest changing it to "scrolling their view"

Reply:
Done